esphome/components/axs15231/touchscreen/axs15231_touchscreen.cpp

```diff
@@ -0,0 +1,63 @@
+#include "axs15231_touchscreen.h"
+
+#include "esphome/core/helpers.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace axs15231 {
+
+static const char *const TAG = "ax15231.touchscreen";
+
+constexpr static const uint8_t AXS_READ_TOUCHPAD[11] = {0xb5, 0xab, 0xa5, 0x5a, 0x0, 0x0, 0x0, 0x8};
+
+#define ERROR_CHECK(err) \
+  if ((err) != i2c::ERROR_OK) { \
+    this->status_set_warning("Failed to communicate"); \
+    return; \
+  }
+
+void AXS15231Touchscreen::setup() {
+  ESP_LOGCONFIG(TAG, "Setting up AXS15231 Touchscreen...");
+  if (this->reset_pin_ != nullptr) {
+    this->reset_pin_->setup();
+    this->reset_pin_->digital_write(false);
+    delay(5);
+    this->reset_pin_->digital_write(true);
+    delay(10);  // NOLINT
```

Comment:
Not needed for only 10ms
```suggestion
    delay(10);
```